src/ucs/async/signal.c

```diff
@@ -101,10 +101,26 @@ ucs_async_signal_set_fd_owner(pid_t dest_tid, int fd)
 #endif
 }
 
+static int
+ucs_async_signal_sys_timer_create_portable(clockid_t clk, struct ksigevent *restrict evp, timer_t *restrict res)
+{
+    int timerid;
+    if (syscall(SYS_timer_create, clk, &evp, &timerid) < 0)
```

Comment:
coding guideline: empty line after variable declarations and open bracket at same line as ```if``` construct:
```
if () {
```
instead of
```
if ()
{
```

Reply:
Ok